src/cpu/aarch64/acl_deconvolution.cpp

```diff
@@ -27,10 +27,18 @@ status_t acl_deconvolution_fwd_t::execute_forward(const exec_ctx_t &ctx) const {
     // concurrent multithreaded access.
     std::lock_guard<std::mutex> _lock {this->mtx};
 
+    const auto scratchpad = ctx.get_scratchpad_grantor();
+
     auto src_base = CTX_IN_MEM(const void *, DNNL_ARG_SRC);
     auto wei_base = CTX_IN_MEM(const void *, DNNL_ARG_WEIGHTS);
     auto bia_base = CTX_IN_MEM(const void *, DNNL_ARG_BIAS);
-    auto dst_base = CTX_OUT_MEM(void *, DNNL_ARG_DST);
+
+    bool use_dst_acc_for_sum = pd()->acl_pd_conf.use_dst_acc_for_sum;
+    // If we have an unfused sum post op, put the result in a scratchpad tensor.
+    // Result will be summed to the dst during acl_post_ops.execute
+    auto dst_base = use_dst_acc_for_sum
+            ? scratchpad.get<void>(memory_tracking::names::key_none)
```

Comment:
`key_none` is not expected to be used as a valid key (here and other spots are concerned).
maybe introduce a generic `key_generic_acc` ?

Reply:
Thank you for the suggestion. That's a good idea. So, I can re-use `key_generic_acc` for all the primitives I am modifying here, correct ?

Reply:
Done.